autogpt_platform/backend/backend/rest.py

```diff
@@ -1,17 +1,16 @@
-from backend.app import run_processes
-from backend.executor import ExecutionScheduler
-from backend.server.rest_api import AgentServer
-
+import uvicorn
+import backend.util.settings
 
 def main():
     """
     Run all the processes required for the AutoGPT-server REST API.
     """
-    run_processes(
-        ExecutionScheduler(),
```

Comment:
The server was still coupled with the Scheduler.

As we are not using it, I've marked the endpoints as depreciated, until such time that we fix how we are handling the scheduler. 

cc: @majdyz